documentation/source/newsfragments/2564.change.rst

```diff
@@ -0,0 +1 @@
+Make :envvar:`COCOTB_REDUCED_LOG_FMT` enabled by default.
```

Comment:
```suggestion
Shorter log lines (configurable with :envvar:`COCOTB_REDUCED_LOG_FMT`) are now the default.
```

Reply:
Can we add a second sentence: "For wider log output, similar to previous cocotb releases, set the COCOTB_REDUCED_LOG_FMT environment variable to 0."

Reply:
```suggestion
Shorter log lines (configurable with :envvar:`COCOTB_REDUCED_LOG_FMT`) are now the default. For wider log output, similar to previous cocotb releases, set the :envvar:`COCOTB_REDUCED_LOG_FMT` environment variable to ``0``.
```